File: js/1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const [a, b, c] = lines[0].split(' ').map(Number);
const maior = (a + b + Math.abs(a - b)) / 2;
const maiorABC = (maior + c + Math.abs(maior - c)) / 2;
console.log(`${maiorABC} eh o maior`);

File: cs/1013.cs
String line = Console.ReadLine();
string[] valor = line.Split(' ');
int a = int.Parse(valor[0]);
int b = int.Parse(valor[1]);
int c = int.Parse(valor[2]);
int maior = (a + b + Math.Abs(a - b)) / 2;
int maiorABC = (maior + c + Math.Abs(maior - c)) / 2;
Console.WriteLine($"{maiorABC} eh o maior");